examples/i18n-routing/next.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  experimental: {
```

Comment:
Let's drop this as-if it's currently stable.

Reply:
Updated